libs/cardano-ledger-api/cardano-ledger-api.cabal

```diff
@@ -0,0 +1,52 @@
+cabal-version:      2.4
+name:               cardano-ledger-api
+version:            0.1.0.0
+synopsis:           Public API for the cardano ledger codebase
+description:
+  This package a public interface into the cardano ledger codebase. It also
+  provides functionality that works for all eras and is useful for downstream
+  packages.
+bug-reports:        https://github.com/input-output-hk/cardano-ledger/issues
+
+license:            Apache-2.0
+author:             IOHK Formal Methods Team
+maintainer:         formal.methods@iohk.io
+copyright:          2021 Input Output (Hong Kong) Ltd.
+
+source-repository head
+  type:     git
+  location: https://github.com/input-output-hk/cardano-ledger
+  subdir:   libs/cardano-ledger-api
+
+common base
+  build-depends:
+    base >= 4.12 && < 4.15
+
+common project-config
+  default-language: Haskell2010
+
+  ghc-options:
+    -Wall
+    -Wcompat
+    -Wincomplete-record-updates
+    -Wincomplete-uni-patterns
+    -Wredundant-constraints
+    -Wpartial-fields
+    -Wunused-packages
+
+library
+  import:
+    base, project-config
+
+  hs-source-dirs: src
+
+  exposed-modules:
+    Cardano.Ledger.Api.Tx
+    Cardano.Ledger.Api.Tx.Out
+
+  build-depends:
+    cardano-ledger-core,
+    cardano-ledger-alonzo,
+    cardano-ledger-babbage,
+    cardano-ledger-shelley,
+    microlens,
```

Comment:
Also not  important nitpick: a new line might be nice here

Reply:
Good nitpick. Done